src/utilities/suggestionList.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/**
+ * Given a JavaScript value and a GraphQL type, determine if the value will be
+ * accepted for that type. This is primarily useful for validating the
+ * runtime values of query variables.
+ */
+export function suggestionList(
+    input: string,
+    options: Array<string>
+  ): Array<string> {
+  let i;
+  const d = {};
+  const oLength = options.length;
+  for (i = 0; i < oLength; i++) {
+    d[options[i]] = lexicalDistance(input, options[i]);
+  }
+  const result = options.slice();
+  return result.sort((a , b) => d[a] - d[b]);
```

Comment:
Maybe instead of or in addition to just sorting them all, we should also filter them down to those with a very low edit distance. Some of the suggestions in the texts seem like they're unlikely to be typos.